src/Endpoint.js

```diff
@@ -10,18 +10,21 @@
 
 // Node dependencies
 const fs = require('fs');
-
-// require extension to load velocity templates
-require.extensions['.vm'] = (module, filename) => {
-  module.exports = fs.readFileSync(filename, 'utf8');
-};
+const { resolve } = require('path');
```

Comment:
Please use `path.resolve` as `resolve` can refer to a Promise. Also for nodejs core modules we use the convention module.method instead of just method.